scss/general.scss

```diff
@@ -5,6 +5,11 @@
 	box-sizing: border-box;
 }
 
+a[href^="http://"],
+a[href^="https://"] {
+	cursor: pointer; // Make all web links use pointer cursor
+}
```

Comment:
This addition is to override the rule set right above this change:

```
*, 
*:before, 
*:after {
	cursor: default; // Make everything use default cursor
	box-sizing: border-box;
}
```

This was originally added because of a recommendation for building Electron apps, because native apps don't have pointer cursors at all. _Except_ for web links that open in a browser, those should use the pointer.

I'm not sure what web elements use the pointer cursor, but we could just do this:

```
button {
cursor: default;
}
```